.ci/setup.sh

```diff
@@ -135,7 +135,7 @@ install_extra_tools() {
 		echo "Install Kubernetes" &&
 		bash -f "${cidir}/install_kubernetes.sh" &&
 		[ "${CRI_CONTAINERD}" = "yes" ] &&
-		bash -f "${cidir}/configure_containerd_for_kubernetes.sh" ||
+		bash -f "${cidir}/configure_containerd_for_kubernetes.sh" &&
```

Comment:
Hi @Jakob-Naucke , good catch!

Note: this `install_extra_tools()` has some unnecessary logical statements as the script sets `errexit`. It deserves a refactor. :)

Reply:
@wainersm would that be as simple as removing all the `&& echo {successful} || die {unsuccessful}`s?

Reply:
yes @Jakob-Naucke , if any of those bash evocations fails then the whole script fails (i.e. don't need to call `die`, unless it provides useful information...don't know)